deepspeed/monitor/monitor.py

```diff
@@ -27,14 +27,14 @@ def __init__(self, monitor_config):
         self.tb_monitor = None
         self.wandb_monitor = None
         self.csv_monitor = None
-        self.enabled = monitor_config.tensorboard_enabled or monitor_config.csv_monitor_enabled or monitor_config.wandb_enabled
+        self.enabled = monitor_config.tensorboard.enabled or monitor_config.csv_monitor.enabled or monitor_config.wandb.enabled
```

Comment:
Could this be simplified by adding an `enabled()` function to `DeepSpeedMonitorConfig` to the details?